### PR TITLE
make test clean, build, and reset dist

### PR DESCRIPTION
**Pull request by buzzdecafe**

### CODE REVIEW

**Comment by buzzdecafe** [comment]
inent it to make it easier to run `npm test`. see https://github.com/ramda/ramda/issues/1152

File: package.json [hunk 1]
```diff
@@ -37,11 +37,12 @@
...
     "build": "make && make dist/ramda.min.js",
-    "clean": "rimraf dist",
+    "clean": "rimraf dist/*",
     "jscs": "jscs scripts/bookmarklet scripts/build **/*.js **/**/*.js",
     "jshint": "jshint scripts/bookmarklet scripts/build **/*.js **/**/*.js",
-    "pretest": "npm run jshint && npm run jscs",
-    "test": "node scripts/testfiles | node_modules/.bin/mocha --reporter spec"
+    "pretest": "npm run jshint && npm run jscs && npm run clean && npm run build",
+    "test": "node scripts/testfiles | node_modules/.bin/mocha --reporter spec",
+    "posttest": "git checkout -- dist/ramda*.js"
```
Comment (davidchambers) [inline package.json]: We could use `git checkout -- dist` here, I think.

**Comment by davidchambers** [comment]
:deciduous_tree:

File: package.json (hunk 1 above)
Comment (buzzdecafe) [inline package.json]: curses! almost snuck one by you without needing `commit --amend`. oh well.

**Comment by benjycui** [comment]
:+1:

**Comment by CrossEye** [comment]
:herb:

**Pull request merged**